### ISSUE TITLE
Estimated time becomes very wrong when starts adding 2 second sleeping

### ISSUE THREAD

**Issue by kangarooo** [imapsync/imapsync#374]
Estimated time becomes very wrong when starts adding 2 seconds sleeping
Need to add it calculating messages left * 2 seconds

Can you point where is it in code so can make a fix?

https://github.com/imapsync/imapsync/search?q=eta

![image](https://user-images.githubusercontent.com/73187/210047539-e020d93c-7f7a-4934-b0ed-add89a3fbcd6.png)

**Comment by gilleslamiral** [comment]
> Estimated time becomes very wrong when starts adding 2 seconds sleeping

Yes, but it gets better as time passes, adjusting to the new mean.

> Need to add it calculating messages left * 2 seconds

It will be better as the "2 seconds sleeping" starts but worse as time passes.

> Can you point where is it in code so can make a fix?

Search for:
sub eta
{
